cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):

```

Comment:
You could use a `ModelSerializer` to automate some operations done upon the model.